scripts/LocationList.js

```diff
@@ -20,32 +20,36 @@ function getNumberOfGuestsForParkArea(locationId) {
 // Function to get the services for a specific park area
 function getLocationServiceMatches(locationId, allLocationServices, allServices) {
```

Comment:
We can just import getLocationServiceMatches from pairings.js - we don't need separate logic here